codexctl/__init__.py

```
@@ -1,733 +1,364 @@
-            )
-            while True:
-                server_host = input(
-                    "\nPlease enter your IP for the network the device is connected to: "
+        ### Mounting functionalities
+        elif function in ("extract", "mount"):
+            try:
+                from .analysis import get_update_image
+            except ImportError:
+                raise ImportError(
+                    "remarkable_update_image is required for analysis. Please install it!"
                 )
```

Ruff (0.8.0)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)